src/data/sql/brp/create.gezagsverhoudingen.sql

```diff
@@ -0,0 +1,83 @@
+CREATE TABLE brp_prep.gezagsverhoudingen AS
+
+  SELECT
+    gv."BSN"::varchar                                                                AS burgerservicenummer,
+    gv."Anummer"::varchar                                                            AS anummer,
+    gv."IndGezagMinderjarigeOms"::varchar                                            AS indicatie_gezag_minderjarige,
+    gv."IndCurateleRegisterOms"                                                      AS indicatie_curatele_register, -- TODO: Check this
+    gv."GemeenteOntleningCode"::varchar                                              AS gemeente_gezagsverhouding_document,
+    CASE -- datum ontlening gezagsverhouding
+      WHEN gv."DatumOntlening" IS NULL THEN NULL
+      WHEN gv."DatumOntlening" = '0'
+        OR gv."DatumOntlening" = '00000000' THEN '0000-00-00'
```

Comment:
But now you can simplify this, right? No need to check for "00000000" here, and check for != "00000000" below, as "00000000" has length 8, so that will return "0000-00-00" anyway.

Reply:
The datum format is not definitive. we wait for the answer about the question what to doe if we get a "0". We should not return "0000-00-00" as a valid date